library/include/borealis/views/button.hpp

```diff
@@ -163,6 +163,17 @@ class Button : public Box
      */
     void setState(ButtonState state);
 
+
+    /**
+     * Sets the text of the button. 
+     */
+    void setText(std::string text);
+
+    /**
+     * Gets the text of the button and returns it
```

Comment:
"gets" is a synonym of "return", there is no need to say "get and return" 😛 

"Returns the text of the button" is fine

Reply:
Fixed that. I probably should pay more attention to my grammar.